index.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
     'plugin:unicorn/recommended',
     // Use native JS instead of lodash
     'plugin:you-dont-need-lodash-underscore/compatible',
+    'prettier',
```

Comment:
Any reason not to use `'plugin:prettier/recommended'` instead?

Reply:
Could we add a `.prettierrc` maybe? Or find a way to distribute a set configuration?
Example:
```
{
  "parser": "babel",
  "semi": true,
  "singleQuote": true,
  "bracketSpacing": true,
  "trailingComma": "all",
  "useTabs": false
}
```

Reply:
It does seems like `'plugin:prettier/recommended'` would be a good start since `eslint-config-amex` enables so many rules, you would want an additive approach.

When I run the `eslint-config-prettier` check utility, at least `'prettier/react'` and `'prettier/unicorn'` should be added (possibly `'pretter/standard'`, not sure):

```sh
$ ./node_modules/.bin/eslint --print-config src/index.js | npx eslint-config-prettier
npx: installed 2 in 3.627s
The following rules are unnecessary or might conflict with Prettier:

- array-bracket-spacing
- arrow-parens
- arrow-spacing
- block-spacing
- brace-style
- comma-dangle
- comma-spacing
- comma-style
- computed-property-spacing
- dot-location
- eol-last
- func-call-spacing
- generator-star-spacing
- implicit-arrow-linebreak
- indent
- jsx-quotes
- key-spacing
- keyword-spacing
- linebreak-style
- new-parens
- newline-per-chained-call
- no-extra-parens
- no-extra-semi
- no-floating-decimal
- no-mixed-spaces-and-tabs
- no-multi-spaces
- no-multiple-empty-lines
- no-spaced-func
- no-trailing-spaces
- no-whitespace-before-property
- nonblock-statement-body-position
- object-curly-newline
- object-curly-spacing
- object-property-newline
- one-var-declaration-per-line
- operator-linebreak
- padded-blocks
- quote-props
- react/jsx-closing-bracket-location
- react/jsx-closing-tag-location
- react/jsx-curly-newline
- react/jsx-curly-spacing
- react/jsx-equals-spacing
- react/jsx-first-prop-new-line
- react/jsx-indent
- react/jsx-indent-props
- react/jsx-max-props-per-line
- react/jsx-props-no-multi-spaces
- react/jsx-tag-spacing
- react/jsx-wrap-multilines
- rest-spread-spacing
- semi
- semi-spacing
- semi-style
- space-before-blocks
- space-before-function-paren
- space-in-parens
- space-infix-ops
- space-unary-ops
- switch-colon-spacing
- template-curly-spacing
- template-tag-spacing
- unicode-bom
- unicorn/no-nested-ternary
- unicorn/number-literal-case
- wrap-iife
- yield-star-spacing

The following rules are enabled with options that might conflict with Prettier. See:
https://github.com/prettier/eslint-config-prettier#special-rules

- curly
- no-confusing-arrow

The following rules are enabled but cannot be automatically checked. See:
https://github.com/prettier/eslint-config-prettier#special-rules

- arrow-body-style
- max-len
- no-mixed-operators
- no-tabs
- no-unexpected-multiline
- prefer-arrow-callback
- quotes

```

If I add the `react` and `unicorn` plugins with `'plugin:prettier/recommend'`, it drops down to nothing:

```json
  "extends": [
    "amex",
    "plugin:prettier/recommended",
    "prettier/react",
    "prettier/unicorn"
  ]
```

```sh
$ ./node_modules/.bin/eslint --print-config src/index.js | npx eslint-config-prettier
npx: installed 2 in 2.75s
No rules that are unnecessary or conflict with Prettier were found.
```